gatsby/content/blog/2022/04/2022-04-05-synapse-1.56.mdx

```diff
@@ -0,0 +1,82 @@
+---
+date: '2022-04-05'
+title: Synapse 1.56 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+Happy release day everyone! [Synapse
+1.56](https://github.com/matrix-org/synapse/releases/tag/v1.56.0) is out! Let's
+have a look at what's new in this release.
+
+## Additional restrictions around registration
+
+One common scenario of abuse we have seen on Matrix over the years is attackers
```

Comment:
```suggestion
One common mode of abuse we have seen on Matrix over the years is attackers
```

Reply:
(actually backed it out because I think "scenario" fits best here)

Reply:
'scenario of abuse' kind of grates on me and sounds unidiomatic, 'mode of abuse' sounds better, but not too fussed I suppose

Reply:
Reworded into "abuse scenario" which sounds more correct to me